### ISSUE TITLE
venv for Ansible and Python3 with AWX build 4.0.0

### ISSUE THREAD

**Issue by cbutler65** [ansible/awx#4015]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - API
 - UI

##### SUMMARY
/usr/bin/awx-manage inventory_import in a playbook fails 

Cause: AWX 4.0.0 has pre-installed venv for Python3(AWX) and Python2(Ansible), but requires venv for Python3(Ansible). 

I have installed the venv for Python3(Ansible) and this fixes the problem but it should be pre-installed to enable std functionality. so I don't have to manually install on awx-task and awx-web images. 

I would like the venv for Python3(Ansible) pre-installed with the images.

##### ENVIRONMENT
* AWX version: 4.0.0
* AWX install method:docker on linux, 
* Ansible version:  2.7.9
* Operating System: RHEL 7

##### STEPS TO REPRODUCE

local_action: command /usr/bin/awx-manage inventory_import --source /tmp/Patching_Inventories_updates/"{{ Inventory_Name }}".txt --inventory-name "{{ Inventory_Name }}"

##### EXPECTED RESULTS

inventory is imported

##### ACTUAL RESULTS

Inventory import fails.
After I install the venv for Python3(Ansible) and select the ANSIBLE ENVIRONMENT in the Template, it works.

**Comment by ryanpetrello** [comment]
@cbutler65 so I understand, what you're asking is for awx builds to ship by default with a Python3 ansible virtualenv (instead of a Python2 one)?

**Comment by cbutler65** [comment]
No, I think it should come bundled with all 3, Python3(AWX), Python2(Ansible) and Python3(Ansible).
So that all options are covered and We the users can enable functionality by just choosing the  ANSIBLE ENVIRONMENT from within the UI.
